### ISSUE TITLE
Update focus color to an accessible one. 

### ISSUE THREAD

**Issue by rymorgan** [reactioncommerce/catalyst#68]
Type: **breaking|critical|major|minor**
Minor

**Describe the bug**
Our current focus color doesn't meet accessibility guidelines and needs to be updated to a color that meets contrast guidelines. 

**To Reproduce**
Click tab to get a focus ring. 

**Screenshots**
<img width="1052" alt="Screen Shot 2019-08-19 at 12 08 41 PM" src="https://user-images.githubusercontent.com/20409254/63292685-14f6df00-c27b-11e9-8b3c-9a7f75cd7331.png">

**Additional context**

**Comment by machikoyasuda** [comment]
Just for documentation's sake, this is what our Catalyst Button focus states look like now:

<img width="124" alt="Screen Shot 2019-08-19 at 2 35 56 PM" src="https://user-images.githubusercontent.com/3673236/63301145-b5a2ca00-c28e-11e9-852c-f6404b71618c.png">
<img width="114" alt="Screen Shot 2019-08-19 at 2 35 58 PM" src="https://user-images.githubusercontent.com/3673236/63301146-b63b6080-c28e-11e9-843e-600d0dc4e80a.png">
<img width="154" alt="Screen Shot 2019-08-19 at 2 35 54 PM" src="https://user-images.githubusercontent.com/3673236/63301147-b63b6080-c28e-11e9-83e8-112a10101966.png">

This is what it looks like on some Reaction Admin MUI components:
<img width="99" alt="Screen Shot 2019-08-19 at 2 35 18 PM" src="https://user-images.githubusercontent.com/3673236/63301169-c0f5f580-c28e-11e9-8bc1-1645ce45e98a.png">

**Comment by machikoyasuda** [comment]
For the Select, we chose dark blue for focus, the same color that shows up when you click on the select:

<img width="990" alt="Screen Shot 2019-08-30 at 3 42 37 PM" src="https://user-images.githubusercontent.com/3673236/64054656-f3c8a500-cb3c-11e9-812e-504eb3a176e4.png">

For Tabs, focus looks similar to the Button:
<img width="600" alt="Screen Shot 2019-08-30 at 3 43 01 PM" src="https://user-images.githubusercontent.com/3673236/64054669-0216c100-cb3d-11e9-86cc-9429e7898f8c.png">

For Chip, it looks like this middle one:
<img width="451" alt="Screen Shot 2019-08-30 at 3 43 26 PM" src="https://user-images.githubusercontent.com/3673236/64054676-0e028300-cb3d-11e9-8db0-f90f66bd81df.png">

The Deletable Chip is problematic b/c the Deletable Chip's X button does not have a focus state so it's impossible to close it with a keyboard key.
